Codegen/test/resources/input/TypeCobol/CopyReplace4Colon/CopyReplace4Colon2.rdz.cbl

```diff
@@ -0,0 +1,11 @@
+﻿       IDENTIFICATION DIVISION.
```

Comment:
Test CopyReplace4Colon2.rdz.cbl  is included in test CopyReplace4Colon**3**.rdz.cbl and can be removed.

Reply:
These tests are here to test nested COPY inclusion at depth 2 and 3